dashboard/modules/log/log_manager.py

```diff
@@ -14,6 +14,9 @@
 from ray.util.state.state_manager import StateDataSourceClient
 from ray._private.pydantic_compat import BaseModel
 
+if BaseModel is None:
```

Comment:
I also wonder if we should not suppress import errors in `ray._private.pydantic_compat`

Reply:
cc @edoakes

Reply:
Ah, sorry about that, thanks for the catch.

We should have a test to avoid this being broken in the future!

Reply:
np!